database/migrations/2017_04_22_123157_create_contact_points_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateContactPointsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('contact_points', function (Blueprint $table) {
+            $table->increments('id');
```

Comment:
we don't have id here..so how will we differentiate which contacts belong to who and is s/he a patient, practioner etc